### ISSUE TITLE
About performance

### ISSUE THREAD

**Issue by JWZhao-uestc** [cangozpi/MagicVO#1]
Hello, thanks for your sharing. I trained this model on my device about 100 epochs. But the  performance I got is so bad. 
And I have some questions about the sequence length. In this code, if the batch size is equal to seq length?
Hopefully for your reply.
![Test results visualized 2269](https://user-images.githubusercontent.com/45277090/197916571-d41d36d6-ef3b-4d08-8051-64b656aa9f30.png)
![Training Losses Plot](https://user-images.githubusercontent.com/45277090/197916573-f6f5b0bf-810b-445e-9cd5-829a754f1a14.png)
![Validation Losses Plot](https://user-images.githubusercontent.com/45277090/197916578-b489ce8d-d613-425b-9a2e-a247162c01d3.png)

**Comment by cangozpi** [comment]
Hi,
As far as I can tell the training loss is decreasing so I believe is is learning. It looks like your model is overfitting. You should probably fine tune the model hyperparameters. Hope this helps and if you can find better parameters that work please feel free to share.
Regards,
Can
